lib/Bugsnag.js

```diff
@@ -39,12 +39,9 @@ export class Client {
 
       ErrorUtils.setGlobalHandler((error, isFatal) => {
         if (this.config.autoNotify) {
-          this.notify(error, report => {report.severity = 'error'}, !!NativeClient.notifyBlocking, () => {
-            if (previousHandler) {
-              previousHandler(error, isFatal);
-            }
-          });
-        } else if (previousHandler) {
+            this.notify(error, report => {report.severity = 'error'}, !!NativeClient.notifyBlocking);
```

Comment:
Unfortunately, we can’t just remove the callback block here, as its needed to ensure android reports are written in the event of a native crash. Perhaps a better solution would be to change the initial call check to be 

```js
if (this.config.autoNotify && this.config.shouldNotify())
```

so it will fall through in the event of release stage filtering?